OrbitClientGgp/ClientGgp.cpp

```diff
@@ -235,11 +234,12 @@ void ClientGgp::ProcessTimer(const TimerInfo& timer_info) { timer_infos_.push_ba
 
 // CaptureListener implementation
 void ClientGgp::OnCaptureStarted(
-    int32_t process_id, std::string process_name, std::shared_ptr<Process> process,
+    std::unique_ptr<ProcessData> process,
+    absl::flat_hash_map<std::string, ModuleData*>&& module_map,
     absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> selected_functions,
     TracepointInfoSet selected_tracepoints) {
-  capture_data_ =
-      CaptureData(process_id, process_name, process, selected_functions, selected_tracepoints);
+  capture_data_ = CaptureData(std::move(process), std::move(module_map), selected_functions,
```

Comment:
_process_ and _module_map_ are moved, should _selected_functions_ and _selected_tracepoints_ be moved with `std::move` as well?

Reply:
yes you are right. I changed it.